IdnoPlugins/IndiePub/Pages/MicroPub/Endpoint.php

```diff
@@ -111,14 +111,29 @@ function post()
                         $type = 'photo';
                     } else if ($photo_url = $this->getInput('photo')) {
                         $type      = 'photo';
-                        $success   = $this->uploadFromUrl($photo_url);
+                        $success   = $this->uploadFromUrl('photo', $photo_url);
                         if (!$success) {
                             \Idno\Core\Idno::site()->triggerEvent('indiepub/post/failure', ['page' => $this]);
                             $this->setResponse(500);
                             echo "Failed uploading photo from $photo_url";
                             exit;
                         }
-                    } else if (!empty($name)) {
+                    }
+
+                    if (!empty($_FILES['video'])) {
```

Comment:
Some micropub clients (ownyougram and shpub) upload a video and the preview image at the same time, so having both is actually used. Known throws the photo away right now, but that can be changed in the future.